addons/mail/static/src/messaging/component/follower/follower.scss

```diff
@@ -20,6 +19,8 @@
     align-items: center;
     display: flex;
     flex: 1;
+    padding-inline-start: map-get($spacers, 2);
+    padding-inline-end: map-get($spacers, 2);
```

Comment:
I think for pixel perfect using 3 would actually make the spacing more consistent with the one between the icons?